### PR TITLE
complete level 4

### PR DESCRIPTION
**Pull request by DmitryGhoncharko**

### CODE REVIEW

File: src/main/java/task1/Run.java [hunk 1]
```diff
@@ -18,16 +21,41 @@ public static void main(String[] args) {
...
         System.out.println(midTaxCoefficient / vehicleTypes.length);
 
-        Vehicle[] cars = {new Vehicle(vehicleTypes[0], "Volkswagen Crafter", "5427 AX-7", 2022, 2015, 37600, Color.BLUE, 100.2),
-                new Vehicle(vehicleTypes[0], "Volkswagen Crafter", "6427 AA-7", 2500, 2014, 227010, Color.WHITE, 100.1),
-                new Vehicle(vehicleTypes[0], "Electric Bus E321", "6785 BA-7", 12080, 2019, 20451, Color.GREEN, 100.4),
-                new Vehicle(vehicleTypes[1], "Golf 5", "8682 AX-7", 1200, 2006, 230451, Color.GRAY, 100.0),
-                new Vehicle(vehicleTypes[1], "Tesla Model S 70D", "0001 AA-7", 2200, 2019, 10454, Color.WHITE, 100.0),
-                new Vehicle(vehicleTypes[2], "Hamm HD 12 VV", null, 3000, 2016, 122, Color.YELLOW, 100.20),
-                new Vehicle(vehicleTypes[3], "МТЗ Беларус - 1025.4", "1145 AB-7", 1200, 2020, 109, Color.RED, 100.0)};
+        Vehicle[] cars = {new Vehicle(vehicleTypes[0], "Volkswagen Crafter", "5427 AX-7", 2022, 2015, 37600, Color.BLUE, 100.2,
```
Comment (AntonNovosad) [inline src/main/java/task1/Run.java:24]: You can make individual method to create and fill the array, like createArrayVehicle

Comment (AlexeyKatomachin) [reply src/main/java/task1/Run.java:24]: or call then like init method

Comment (DmitryGhoncharko) [reply src/main/java/task1/Run.java:24]: fixed

**Review by AntonNovosad — APPROVED** [review]

**Comment by AlexeyKatomachin** [comment]
please, rename the folder task1 to autopark

**Comment by AlexeyKatomachin** [comment]
please, separate levels to different PR's

File: src/main/java/autopark/Run.java [hunk 2]
```diff
@@ -50,6 +47,7 @@ private static VehicleType[] initVehicleType() {
         return vehicleTypes;
     }
 
+
```
Comment (AlexeyKatomachin) [inline src/main/java/autopark/Run.java]: please, remove unnecessary empty rows

**Pull request closed without merging**